randovania/monitoring/__init__.py

```diff
@@ -126,25 +123,22 @@ def traces_sampler(sampling_context):
     )
 
 
-def client_init():
-    if not randovania.is_frozen():
-        # TODO: It'd be nice to catch these running from source, but only for unmodified main.
-        return
+def client_init() -> None:
+    _init(False, "client", exclude_server_name=True)
 
-    _init(False, _CLIENT_DEFAULT_URL, exclude_server_name=True)
     sentry_sdk.set_tag("frozen", randovania.is_frozen())
 
     # Ignore the "packet queue is empty, aborting" message
     # It causes a disconnect, but we smoothly reconnect in that case.
     sentry_sdk.integrations.logging.ignore_logger("engineio.client")
 
 
-def server_init(sampling_rate: float):
-    return _init(True, _SERVER_DEFAULT_URL, sampling_rate=sampling_rate)
+def server_init(sampling_rate: float) -> None:
+    return _init(True, "server", sampling_rate=sampling_rate)
 
 
-def bot_init():
-    return _init(False, _BOT_DEFAULT_URL)
+def bot_init() -> None:
+    return _init(False, "bot")
```

Comment:
`bot` needs to be a key in `sentry_url = configuration["sentry_urls"][url_key]` but I cannot find any place where this is set.